Sample/Vungle Sample App/MultiAdViewController.m

```diff
@@ -0,0 +1,310 @@
+//
+//  MultiAdViewController.m
+//  Vungle Sample App
+//
+//  Created by John Mai on 4/6/20.
+//  Copyright © 2020 Jordyn Chuhaloff. All rights reserved.
+//
+
+#import "MultiAdViewController.h"
+#import "Constants.h"
+#import "MultiAdViewTableViewCell.h"
+
+@interface MultiAdViewController () {
+        CGFloat screenHeight;
+        CGFloat screenWidth;
+}
+@property (strong, nonatomic) IBOutlet UIView *view;
+
+@property (nonatomic, strong) VungleSDK *sdk;
+@property MultiAdViewTableViewCell *bannerCell;
```

Comment:
If this view controller is to be used as a table view - we should discuss how the solution can be handled without properties. This is not the best approach.